Extensions/3D/Model3DRuntimeObject.ts

```diff
@@ -74,6 +86,34 @@ namespace gdjs {
     getRenderer(): RuntimeObject3DRenderer {
       return this._renderer;
     }
+
+    _convertMaterialType(
+      materialTypeString: string
+    ): gdjs.Model3DRuntimeObject.MaterialType {
+      if (materialTypeString === 'KeepOriginalMaterial') {
```

Comment:
I'd like the names and internal enums to be closer to the actual technical translation :

- KeepOriginalMaterial (fine)
- StandardMaterialWithoutMetalness
- BasicMaterial

or:

- KeepOriginal (fine)
- StandardWithoutMetalness
- Basic

as you prefer